io_scheduling/disk.py

```diff
@@ -0,0 +1,7 @@
+class Disk:
```

Comment:
add spaces in the proper places `self.track_size=track_size` `self.track_size = track_size`